packages/wonder-blocks-announcer/src/__tests__/util/dom.test.ts

```diff
@@ -0,0 +1,133 @@
+import {screen, waitFor} from "@testing-library/react";
+import {
+    createRegionWrapper,
+    createDuplicateRegions,
+    createRegion,
+    removeMessage,
+} from "../../util/dom";
+
+jest.useFakeTimers();
+jest.spyOn(global, "setTimeout");
+
+describe("Announcer utility functions", () => {
+    describe("createRegionWrapper", () => {
+        test("it creates a polite region wrapper element", () => {
+            const element = createRegionWrapper("polite");
+
+            expect(element.tagName).toBe("DIV");
+            expect(element.id).toEqual("wbAWrap-polite");
+        });
+
+        test("it creates an assertive region wrapper element", () => {
+            const element = createRegionWrapper("assertive");
+
+            expect(element.tagName).toBe("DIV");
+            expect(element.id).toEqual("wbAWrap-assertive");
+        });
+    });
+
+    describe("createDuplicateRegions", () => {
+        test("it creates multiple polite region elements", () => {
+            const wrapper = document.createElement("div");
+            const dictionary = new Map();
+
+            const regionList = createDuplicateRegions(
+                wrapper,
+                "polite",
+                2,
+                dictionary,
+            );
+
+            expect(regionList.length).toBe(2);
+            expect(regionList[0].id).toBe("wbARegion-polite0");
+            expect(regionList[1].id).toBe("wbARegion-polite1");
+            expect(dictionary.size).toBe(2);
+        });
+
+        test("it creates multiple assertive region elements", () => {
+            const wrapper = document.createElement("div");
+            const dictionary = new Map();
+
+            const regionList = createDuplicateRegions(
+                wrapper,
+                "assertive",
+                2,
+                dictionary,
+            );
+
+            expect(regionList.length).toBe(2);
+            expect(regionList[0].id).toBe("wbARegion-assertive0");
+            expect(regionList[1].id).toBe("wbARegion-assertive1");
+            expect(dictionary.size).toBe(2);
+        });
```

Comment:
**suggestion:** I think this test could be dropped as the actual functionality is tested in the previous test.